src/v2/view-builder/internals/BaseForm.js

```diff
@@ -68,6 +68,14 @@ export default Form.extend({
     }
   },
 
+  postRender() {
+    // widget would use info container to display interactive messages that would not be cleared once displayed
+    // but can be replaced by view classes. For eg resend-warning callout should not be cleared upon form submission
+    // if there is any error. Rerender would clear info container.
```

Comment:
Just to clarify this comment, `if there is any error. Rerender would clear info container`, basically the only way this new container would get cleared is upon re-render right? There's no other special logic that would clear this new container, hence that's what makes this somewhat persistent.

Reply:
Updated the comment.